test/e2e_node/environment/setup_host.sh

```diff
@@ -42,38 +42,35 @@ EOF
 fi
 
 # For coreos, disable updates
-if $(sudo systemctl status update-engine &>/dev/null); then
+if sudo systemctl status update-engine &>/dev/null; then
```

Comment:
change reason:
```
In ./test/e2e_node/environment/setup_host.sh line 45:
if $(sudo systemctl status update-engine &>/dev/null); then
   ^-- SC2091: Remove surrounding $() to avoid executing output.
```